backend/app/views/spree/admin/prototypes/_prototypes.html.erb

```diff
@@ -14,7 +14,7 @@
       <tr id="row_<%= prototype.id %>" data-hook="available_row" class="<%= cycle('odd', 'even')%>">
         <td><%= prototype.name %></td>
         <td class="actions">
-          <%= link_to Spree.t(:select), select_admin_prototype_url(prototype), :class => 'ajax button select_properties_from_prototype fa fa-ok' %>
+          <%= link_to Spree.t(:select), select_admin_prototype_url(prototype), class: 'ajax button select_properties_from_prototype fa fa-ok', data: { remote: true } %>
         </td>
```

Comment:
ujs is brilliant in such cases :+1:  

I've had success in the past also adding the below
```
data-disable-with=" &nbsp; <span class='fa fa-icon-spinner fa-lg fa-spin'></span>  &nbsp; "
```
( not sure about the exact class names you can use )